server/src/config.rs

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) ZeroC, Inc.
+
+use crate::Backend;
+use tower_lsp::lsp_types::{ConfigurationItem, DidChangeConfigurationParams, Url};
+
+#[derive(Default, Debug)]
+pub struct SliceConfig {
+    pub reference_urls: Option<Vec<Url>>,
```

Comment:
We dont currently, but it is more correct for it to be an option per the lsp spec. This will be refactored in a future PR though too for multiworkspace support.